e2e/__tests__/__snapshots__/failures.test.js.snap

```diff
@@ -360,8 +360,9 @@ exports[`works with async failures 1`] = `
 
     expect(received).rejects.toEqual()
 
-    Expected received Promise to reject, instead it resolved to value
-      {\\"foo\\": \\"bar\\"}
+    Received promise resolves instead of rejects
+
+    Received value: {\\"foo\\": \\"bar\\"}
```

Comment:
should be "Resolved value: ", or "Received value: Promise<{"foo": "bar"}>"

Reply:
Three set of eyes are better than two sets. What do y’all think about:

```sh
Received promise resolves instead of rejects

Received promise has value: {"foo": "bar"}
```

Reply:
`Received promise resolved to` sounds better to me, and more closely matches the language I've seen used about promises previously. They "resolve" to values, they don't "contain" values

Reply:
Also, maybe "resolved" instead of "resolves" and "rejected" instead of "rejects"? The resolution/rejection of the promise has happened at this point

Reply:
Although I am not so strict on grammar to forbid ending with `to` how about `… to value:`

<img width="1000" alt="instead-1-resolves-false" src="https://user-images.githubusercontent.com/11862657/50542146-0ccf1f80-0b84-11e9-972f-ba6c4843b8af.png">